gucci.go

```diff
@@ -59,6 +68,15 @@ func main() {
 	app.Run(os.Args)
 }
 
+func getTplOpt(cliTplOpt []string) []string {
```

Comment:
Is possible to set a default `flagSetOptLong` arg instead of having a separate function to set the default here?

I think that urfave/cli should allow the option to set a default, no?

Reply:
Indeed, much simpler